$NetBSD: patch-CMakeLists.txt,v 1.2 2012/03/22 11:47:16 drochner Exp $

Increase taglib minor version to be bigger than in the last automake
release.

--- CMakeLists.txt.orig	2012-03-17 10:02:24.000000000 +0000
+++ CMakeLists.txt
@@ -39,7 +39,7 @@ if (WIN32)
 endif (WIN32)
 
 SET(TAGLIB_LIB_MAJOR_VERSION "1")
-SET(TAGLIB_LIB_MINOR_VERSION "7")
+SET(TAGLIB_LIB_MINOR_VERSION "10")
 SET(TAGLIB_LIB_PATCH_VERSION "1")
 
 SET(TAGLIB_LIB_VERSION_STRING "${TAGLIB_LIB_MAJOR_VERSION}.${TAGLIB_LIB_MINOR_VERSION}.${TAGLIB_LIB_PATCH_VERSION}")
